Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackWidget.swift

```diff
@@ -151,6 +58,57 @@ struct SentryUserFeedbackWidget {
             return result
         }
     }
+
+    func setWidget(visible: Bool, animated: Bool) {
+        rootVC.setWidget(visible: visible, animated: animated)
+    }
+
+    class RootViewController: UIViewController, UIAdaptivePresentationControllerDelegate {
+        let defaultWidgetSpacing: CGFloat = 8
+        unowned let button: SentryUserFeedbackWidgetButtonView
+        init(config: SentryUserFeedbackConfiguration, button: SentryUserFeedbackWidgetButtonView) {
+            self.button = button
+            super.init(nibName: nil, bundle: nil)
+
+            view.addSubview(button)
+
+            var constraints = [NSLayoutConstraint]()
+            if config.widgetConfig.location.contains(.bottom) {
+                constraints.append(button.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor, constant: -config.widgetConfig.layoutUIOffset.vertical))
+            }
+            if config.widgetConfig.location.contains(.top) {
+                constraints.append(button.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: config.widgetConfig.layoutUIOffset.vertical))
+            }
+            if config.widgetConfig.location.contains(.trailing) {
+                constraints.append(button.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor, constant: -config.widgetConfig.layoutUIOffset.horizontal))
+            }
+            if config.widgetConfig.location.contains(.leading) {
+                constraints.append(button.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor, constant: config.widgetConfig.layoutUIOffset.horizontal))
+            }
+            NSLayoutConstraint.activate(constraints)
+        }
+
+        required init?(coder: NSCoder) {
+            fatalError("init(coder:) has not been implemented")
```

Comment:
`l`: We should add a comment or change the error message to explain why it wasn't implemented so it does not look like an undocumented TODO.

For example:
```suggestion
            fatalError("init(coder:) is not supported for the Sentry user feedback widget window")
```